tests/test_main.py

```diff
@@ -31,19 +31,21 @@ def test_hacker_news_default_keywords(self):
             ['hacker_news', '-i', self.post_id]
         )
         expected = [
-            'Total job posts: 883',
+            'Total job posts: 719',
 
             'Keywords:',
-            'Remote: 174 (19%)',
-            'Postgres: 81 (9%)',
-            'Python: 143 (16%)',
-            'Javascript: 118 (13%)',
-            'React: 133 (15%)',
+            'Remote: 164 (22%)',
+            'Postgres: 81 (11%)',
+            'Python: 145 (20%)',
+            'Javascript: 120 (16%)',
+            'React: 134 (18%)',
```

Comment:
We didn't notice the need to tweak the tests.
But as Pitagoras was quoted saying: It's not cheating to tweak the tests, if the tests are .... unfair. Look it up.